pythonforandroid/toolchain.py

```diff
@@ -69,7 +78,7 @@ def format(self, record):
                                     # handler and reset the level
     logger.setLevel(logging.INFO)
     logger.touched = True
-    ch = logging.StreamHandler(stdout)
+    ch = logging.StreamHandler(stdout) if sys.stdout.isatty() else logging.NullHandler()
```

Comment:
for the remaining cases of explicit calls to distribute.sh that I have seen, buildozer not only doesn't want the log, but it's actually detrimental to its functionning.  Maybe the log should be preserved, but it has to go somewhere else than stdout.  I am not certain what the best design should be (I am discovering the new code base).